tst/regression/test_suites/advection_outflow/advection_outflow.py

```diff
@@ -50,4 +50,18 @@ def Analyse(self, parameters):
             check_metadata=False,
         )
 
-        return delta == 0
+        try:
+            from phdf import phdf
+        except ModuleNotFoundError:
+            print("Couldn't find module to open Parthenon hdf5 files.")
+            return False
+
+        ## compute the derived var 'manually' and compare to the output derived var
+        data_filename = "outflow.out0.final.phdf"
+        data_file = phdf(data_filename)
+        q = data_file.Get("advected")[0]
```

Comment:
@pgrete The test fails on this line with this error:
```
            ERROR in phdf.Get: Unable to read advected from file outflow.out0.final.phdf
```
I can't figure out why. Is there something obvious I've missed here?

Reply:
Maybe `advected` needs to be capitalized to `Advected`? Wasn't that changed recently? It caused a failure in the yt frontend

Reply:
That did not help:
```

            ERROR in phdf.Get: Unable to read Advected from file outflow.out0.final.phdf
            
Traceback (most recent call last):
  File "/Users/benwibking/parthenon/tst/regression/run_test.py", line 204, in <module>
    main(**vars(args))
  File "/Users/benwibking/parthenon/tst/regression/run_test.py", line 88, in main
    test_result = test_manager.Analyse()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/benwibking/parthenon/tst/regression/utils/test_case.py", line 278, in Analyse
    test_pass = self.test_case.Analyse(self.parameters)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/benwibking/parthenon/tst/regression/test_suites/advection_outflow/../../test_suites/advection_outflow/advection_outflow.py", line 62, in Analyse
    q = data_file.Get("Advected")[0]
        ~~~~~~~~~~~~~~~~~~~~~~~~~^^^
TypeError: 'NoneType' object is not subscriptable

```

Reply:
The same thing is done here, using `"advected"`: https://github.com/parthenon-hpc-lab/parthenon/blob/a97831ac78ca31eed09b1a2daf6815042efc6f46/tst/regression/test_suites/output_hdf5/output_hdf5.py#L175

Reply:
I will try moving this to the `output_hdf5` test.

Reply:
maybe the variable was not parsed correctly (see comment below) and thus didn't end up in the output file

Reply:
That was it.

Reply:
I'll re-create the gold files with it included.